lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -1156,8 +1156,13 @@ def create_computed_questionnaire_response(record: dict, patient_reference: dict
     For example, a computed question captures the participant's age
     on the date of the encounter.
     """
-
-    record['age'] = relativedelta(encounter_date, birthdate).years
+    # A birthdate of None will return a falsy relativedelta() object
+    delta = relativedelta(encounter_date, birthdate)
+    if not delta:
+        age = None
+    else:
+        age = delta.years
+    record['age'] = age
```

Comment:
Oh second thought I think it might be better to avoid entering age altogether?

Reply:
My thoughts being the usual "null values is a violation of the FHIR docs". But I'm not quite sure how this would affect QuestionnaireResponse itemAnswer calculation and I don't really feel like getting too deeply into this at 8AM on a Saturday morning.